Source/msg.cpp

```diff
@@ -863,7 +863,7 @@ void NetSendCmd(bool bHiPri, _cmd_id bCmd)
 		NetSendLoPri(myplr, (BYTE *)&cmd, sizeof(cmd));
 }
 
-void NetSendCmdGolem(BYTE mx, BYTE my, BYTE dir, BYTE menemy, int hp, BYTE cl)
+void NetSendCmdGolem(BYTE mx, BYTE my, direction dir, BYTE menemy, int hp, BYTE cl)

```

Comment:
TCmdGolem._mdir is defiend as `Uint8`: static_cast or change to enum or leave it?

Reply:
forget it. didn't read far enough :rofl: